package.json

```diff
@@ -88,10 +87,6 @@
     "yarn": ">=3"
   },
   "config": {
-    "ui-scripts": {
-      "npm_scope": "@instructure:registry=https://registry.npmjs.org/",
-      "changelog_url": "https://instructure.design/#CHANGELOG"
-    },
```

Comment:
this is the config of ui-scripts. changelog_url is unused now, I've moved the other to a constant where its used.